pkgs/applications/networking/instant-messengers/rambox-pro/default.nix

```
@@ -0,0 +1,46 @@
+{ autoPatchelfHook, electron, fetchurl, makeDesktopItem, makeWrapper, nodePackages, nss, stdenv, xdg_utils, xorg }:
+
+stdenv.mkDerivation rec {
+  pname = "rambox-pro-${version}";
```

That should be:
```suggestion
  pname = "rambox-pro";
```
Then nix will automatically define package name as being `rambox-pro-1.1.2` (ie `${pname}-${version}`